jobs/README.md

```diff
@@ -9,11 +9,13 @@ __Be advised that Job Openings older than 3 months will be removed for the sake
 Come work for a team that works on high-performance telecoms infrastructure systems.
 We value technical expertise, kindness and a sense of humour.
 
-* PubDate: 09/05/2017
+* PubDate: 2017-05-09
```

Comment:
I always make a comment out of this, we are in GR and the date format in GR is day/month/year.  Anything else is computer language, this job post is to be read by humans, not parsed by a computer. never the less I will merge